spartanstay/frontend/public/signup.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+
+    <title>React App</title>
+  
+    <a href='index.html'><h1>Home</h1></a>
+    <a href="signup.html"><h1>Sign-Up</h1></a>
+  </head>
+
+  <body>
+
+    <label for="fName"><b>First Name</b></label>
```

Comment:
This is a great start, but we need this html in React components to be able to attach hooks to it. Without the hooks, it will be much harder to get data from the frontend to the backend.